package/MDAnalysis/visualization/streamlines_3D.py

```diff
@@ -34,7 +34,7 @@
 '''
 from __future__ import division, absolute_import
 import six
-from six.moves import range
```

Comment:
I think here you'll need to do `list(zip(` again, else you'll end up appending generators to this list

https://github.com/kain88-de/mdanalysis/blob/8f35bebf03613601e74d12507be0df7f293b1ce6/package/MDAnalysis/visualization/streamlines_3D.py#L101